tests/registry.cc

```diff
@@ -50,6 +57,61 @@ test_FloxFlakeInputRegistry0()
 }
 
 
+/* -------------------------------------------------------------------------- */
+
+  bool
+test_RegistryFileMixinHappyPath()
+{
+  using namespace flox;
+
+  flox::command::RegistryFileMixin rfm;
+  rfm.setRegistryPath( TEST_DATA_DIR "/registry/registry0.json" );
+  RegistryRaw regRaw = rfm.getRegistryRaw();
+
+  return true;
```

Comment:
question: do we have coverage somewhere already that `regRaw` gets deserialized as we expect it to?

Reply:
I feel like there has to be... let me look for it

Reply:
As far as I can tell it's just `test_FloxFlakeInputRegistry0` testing that by way of using it to construct a `Registry`...